content/telegraf/v1.22/configuration.md

```diff
@@ -0,0 +1,474 @@
+---
+title: Configuration options
+description: Overview of the Telegraf configuration file, enabling plugins, and setting environment variables.
+menu:
+  telegraf_1_22_ref:
+
+    name: Configuration options
+    weight: 20
+
+---
+
+The Telegraf configuration file (`telegraf.conf`) lists all available Telegraf plugins. See the current version here: [telegraf.conf](https://github.com/influxdata/telegraf/blob/master/etc/telegraf.conf).
+
+> To quickly get started with Telegraf, see [Get started](/{{< latest "telegraf" >}}/get_started/).
```

Comment:
@noramullen1 @lwandzura We shouldn't use the `{{< latest >}}` shortcode when linking to content in the same product. Only use it when linking to other product content. When we release Telegraft 1.23, all the latest links in 1.22 would point to 1.23 and shift the version context when users click on the link. We don't want that.

Reply:
oh duh, covid brain